### ISSUE TITLE
Potential bug in score method for sklearn API

### ISSUE THREAD

**Issue by vdrao** [stanfordmlgroup/ngboost#218]
Hello,

Thank you for NGBoost! I think there may be a bug in the sklearn score API in the [main NGBoost class](https://github.com/stanfordmlgroup/ngboost/blob/master/ngboost/ngboost.py) (lines 317-318):

```
    def score(self, X, Y):  # for sklearn
        return self.Manifold(self.pred_dist(X)._params).total_score(Y)
```

Sklearn hyperparameter tuning functions like `RandomizedSearchCV` expect that better models are assigned higher (more positive) scores. However, as far as I can see, the `total_score` method above actually computes the total _log loss_, which means that smaller values correspond to better models. Should the score instead be the negative log loss?

**Comment by alejandroschuler** [comment]
Hi @vdrao, the method you mention actually calculates the _negative_ log likelihood (when using the `LogScore` scoring rule). The correct sign is enforced in the definition of the score method for each distribution, [e.g. for the normal](https://github.com/stanfordmlgroup/ngboost/blob/4c2b11d37ae89d74a5bc5d6119ffec598f318470/ngboost/distns/normal.py#L12).
